pwiz_tools/Skyline/Model/Export.cs

```diff
@@ -4257,6 +4256,8 @@ protected override void WriteTransition(TextWriter writer, int fileNumber, Pepti
             TransitionGroupDocNode nodeTranGroup, TransitionGroupDocNode nodeTranGroupPrimary, TransitionDocNode nodeTran,
             int step)
         {
+            writer.Write(GetCompound(nodePep, nodeTranGroup, true));

```

Comment:
Do we need to worry about what if the compound name contains commas?
Would it work to put quotes around the value, or should the comma get replaced with a safe character?

Reply:
I don't know that we have nailed this everywhere in this file, but it seems like only the small molecule value:

nodeGroup.CustomMolecule.InvariantName

Could have a comma. Or maybe a custom isotope label type? It seems like the solution would be to use:

text.ToDsvField(FieldSeparator, FieldSeparatorReplacement)

As gets used in ThermoMassListExporter and WatersMassListExporter:

            writer.WriteDsvField(GetCompound(nodePep, nodeTranGroup), FieldSeparator, FieldSeparatorReplacement);

Though, If I put this in the base implementation of GetCompound() then, the above could become just writer.Write().

Reply:
This doesn't rely on the vendor software being able to deal with the full flexibility of CSV format.

Reply:
Okay, I added code for separator replacement to GetCompound(). I am going to wait for Brian P's return to add a test for this, because I don't see an easy way to change the name of a molecule in the SrmDocument tree. The CustomMolecule is still part of the Peptide which is the ID class for PeptideDocNode. So, changing it seems to require a lot of work. Though, I may be missing something.